docs/portal_config.md

```diff
@@ -572,6 +572,28 @@ Below is an example, with inline comments describing what each JSON block config
         }
       ]
     },
+    "detailView": {
```

Comment:
should mention that `detailView` will take precedence over other `studyPageFields` configs